packages/paste-website/stories/SidebarNavigation.stories.tsx

```diff
@@ -8,79 +9,87 @@ import {SidebarDisclosureButton} from '../src/components/site-wrapper/sidebar/si
 import {SidebarDisclosureContent} from '../src/components/site-wrapper/sidebar/sidebar-disclosure/SidebarDisclosureContent';
 
 export const ComplexSidebar = (): React.ReactNode => (
-  <Box as="ul" listStyleType="none">
-    <SidebarAnchor level={0} to="https://www.google.com">
-      Level 0 anchor
-    </SidebarAnchor>
-    <SidebarAnchor level={0} to="https://www.google.com">
-      Level 0 anchor
-    </SidebarAnchor>
-    <SidebarDisclosure categoryRoute="/introduction" visible>
-      <SidebarDisclosureButton level={0}>Level 0 disclosure</SidebarDisclosureButton>
-      <SidebarDisclosureContent>
-        <SidebarAnchor level={1} to="https://www.google.com">
-          Level 1 anchor
-        </SidebarAnchor>
-        <SidebarDisclosure categoryRoute="/introduction/for-designers" visible>
-          <SidebarDisclosureButton level={1}>Level 1 disclosure</SidebarDisclosureButton>
-          <SidebarDisclosureContent>
-            <SidebarAnchor level={2} to="https://www.google.com">
-              Level 2 anchor
-            </SidebarAnchor>
-            <SidebarAnchor level={2} to="https://www.google.com">
-              Level 2 anchor
-            </SidebarAnchor>
-          </SidebarDisclosureContent>
-        </SidebarDisclosure>
-      </SidebarDisclosureContent>
-    </SidebarDisclosure>
-  </Box>
+  <Router>
```

Comment:
issue: I tried to open storybook from my browser and got an error for these stories that looked like this:
<img width="1171" alt="image" src="https://user-images.githubusercontent.com/77117846/154768175-06a2e22d-6c26-45f4-9923-0176681edce9.png">
I'm confused by the error since it seems to be saying that there's another, bigger error?